src/GayatriG/Assignment2Ex5.java

```diff
@@ -0,0 +1,28 @@
+//5)Write a java program to find second largest element in an array of integers? 
+
+package GayatriG;
+
+public class Assignment2Ex5 {
+	void secondLargestNum(int[] input) {
+		int largestNum = 0;
+		int secLargestNum = 0;
+		for (int i = 0; i < input.length; i++) {
+			if (input[i] > largestNum) {
+				largestNum = input[i];
+			}
+		}
+		for (int i = 0; i < input.length; i++) {
+			if (input[i] > secLargestNum && input[i] < largestNum) {
+				secLargestNum = input[i];
+			}
+		}
+		System.out.println(secLargestNum + " is the second largest number in the given array");
+	}
+
+	public static void main(String[] args) {
+		int[] arr = { 2, 6, 98, 34, 66, 79, 4 };
```

Comment:
Hi @GayatriGAug19 
Here for this array int[] arr = { 2, 6, 98, 34, 66, 79, 4,100 };
expected output is not generated...
please check and debug code...

Reply:
Hi @Harsh9846 , I ran the code with the given input and i am getting expected output as"98 is the second largest number in the given array"